lib/runner/clientmanager.js

```diff
@@ -62,6 +62,10 @@ ClientManager.prototype.results = function(type, value) {
   return this;
 };
 
+ClientManager.prototype.clearResult = function() {
+  return this['@client'].clearResult.call(this['@client']);
```

Comment:
Yeah I was copying the pattern here: https://github.com/beatfactor/nightwatch/blob/master/lib/runner/clientmanager.js#L70

But yeah seems like I can simplify that..